offchain/rollups-events/src/broker/mod.rs

```diff
@@ -155,15 +151,15 @@ impl Broker {
         Ok(event_id)
     }
 
-    /// Peek at the end of the stream
+    /// Read event at the end of the stream
     /// This function doesn't block; if there is no event in the stream it returns None.
     #[tracing::instrument(level = "trace", skip_all)]
-    pub async fn peek_latest<S: BrokerStream>(
+    pub async fn read_latest<S: BrokerStream>(
```

Comment:
Since we're just checking what is the latest event, I'd rather keep the original name:
```suggestion
    pub async fn peek_latest<S: BrokerStream>(
```